internal/backend/crypto/age/commands.go

```diff
@@ -0,0 +1,94 @@
+package age
+
+import (
+	"fmt"
+
+	"filippo.io/age"
+	"github.com/gopasspw/gopass/pkg/ctxutil"
+	"github.com/urfave/cli/v2"
+)
+
+func (l loader) Commands() []*cli.Command {
+	return []*cli.Command{
+		{
+			Name:   name,
+			Hidden: true,
+			Usage:  "age commands",
+			Description: "" +
+				"TODO",
+			Subcommands: []*cli.Command{
+				{
+					Name:  "identities",
+					Usage: "List identities",
+					Description: "" +
+						"List identities",
+					Action: func(c *cli.Context) error {
+						ctx := ctxutil.WithGlobalFlags(c)
+						a, err := New()
+						if err != nil {
+							return err
+						}
+
+						ids, err := a.IdentityRecipients(ctx)
+						if err != nil {
+							return err
+						}
+
+						for _, id := range recipientsToBech32(ids) {
+							fmt.Println(id)
+						}
+
+						return nil
+					},
+					Subcommands: []*cli.Command{
+						{
+							Name:  "add",
+							Usage: "Add an identity",
+							Description: "" +
+								"Add an identity",
+							Action: func(c *cli.Context) error {
+								ctx := ctxutil.WithGlobalFlags(c)
+								a, err := New()
+								if err != nil {
+									return err
+								}
+
+								if err := a.GenerateIdentity(ctx, "", "", ""); err != nil {
+									return err
+								}
+
+								return nil
+							},
+						},
+						{
+							Name:  "remove",
+							Usage: "Remove an identity",
+							Description: "" +
+								"Remove an identity",
+							Action: func(c *cli.Context) error {
+								ctx := ctxutil.WithGlobalFlags(c)
+								a, err := New()
+								if err != nil {
+									return err
+								}
+								victim := c.Args().First()
+
+								ids, _ := a.Identities(ctx)
+								newIds := make([]string, 0, len(ids))
+
+								for _, id := range ids {
+									if x, ok := id.(*age.X25519Identity); ok && x.Recipient().String() == victim {
```

Comment:
I've probably missed it in an earlier PR, but didn't we use to support more than just X25519 identities? 

Is that going to be "backward compatible" with SSH identities that we used to support and that could be using P-256 for instance? Or even worse, `age` is compatible with `ssh-rsa` IIRC.

Reply:
added a comment to clarify. let me know if that doesn't explain it well enough.